<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StcTablesCreator</class>
 <widget class="QDialog" name="StcTablesCreator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Table Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="5">
    <widget class="QTableWidget" name="tableWidget">
     <property name="editTriggers">
      <set>QAbstractItemView::EditTrigger::DoubleClicked|QAbstractItemView::EditTrigger::EditKeyPressed</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectionBehavior::SelectItems</enum>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>120</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>24</number>
     </attribute>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="addColumnButton">
     <property name="toolTip">
      <string>Add column</string>
     </property>
     <property name="text">
      <string>+ Column</string>
     </property>
     <property name="icon">
      <iconset theme="list-add"/>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QCheckBox" name="extendedCheckBox">
     <property name="text">
      <string>extended</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="addRowButton">
     <property name="toolTip">
      <string>Add row</string>
     </property>
     <property name="text">
      <string>+ Row</string>
     </property>
     <property name="icon">
      <iconset theme="list-add"/>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QCheckBox" name="headerCheckBox">
     <property name="text">
      <string>with header</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
